### PR TITLE
Final test

### PR DESCRIPTION
**Pull request by mkrtchyan-t**
Added comments and fixed percentages in `print_stat.sh` file.
```
disk=$(df -h | head -4 | tail -1 | tr -s " " | cut -d " " -f 5)
```
something like
> CPU Usage: 1.4 %

### CODE REVIEW

**Review by sargsyan1986 — APPROVED** [review]
Maladec ... shat lava ... qez mi me& like !

**Review by Frida199 — APPROVED** [review]

File: nginx_install.sh [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
+# updating, installing and starting nginx and required libraries
 apt update -y
-
 sudo apt install nginx -y
-
-ufw app list
-
 ufw allow 'Nginx Full'
-
 systemctl start nginx
```
Comment (mkrtchyan-t) [inline nginx_install.sh:7]: should use
```
systemctl restart nginx
```

File: nginx_install.sh [hunk 2]
```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 
+# updating, installing and starting nginx and required libraries
 apt update -y
-
 sudo apt install nginx -y
-
-ufw app list
-
 ufw allow 'Nginx Full'
```
Comment (mkrtchyan-t) [inline nginx_install.sh:6]: add a comment describing what is this for

File: nginx_install.sh [hunk 3]
```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 
+# updating, installing and starting nginx and required libraries
 apt update -y
-
 sudo apt install nginx -y
```
Comment (mkrtchyan-t) [inline nginx_install.sh:5]: before installing check if nginx is already installed and remove the `sudo`

File: nginx_conf.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Updating and installing required commands
+apt update  -y
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:4]: we don't need this, as if everything is updated in `nginx-install.sh` file

File: nginx_conf.sh [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+apt install wget -y
+apt install curl -y
+
+# initiating variables
+URL=https://transfer.sh/QX7UuX/bootcamp_aca.conf
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:9]: change the variable name to `url` and add single parathesis like
```
url='https://transfer.sh/QX7UuX/bootcamp_aca.conf'
```

File: nginx_conf.sh [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+response=$(curl --write-out %{http_code} --silent --output /dev/null $URL)
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:10]: change to
```
response=$(curl --write-out %{http_code} --silent --output /dev/null $usr)
```

File: nginx_conf.sh [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+# Checking if the given URL works and if we already have one
+if [[ $response = 200 ]]; then
+	wget $URL
+fi
+
+# checking if we already have the copy as our nginx default conf
+deff=$(diff bootcamp_aca.conf /etc/nginx/sites-enabled/default)
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:18]: delete

File: nginx_conf.sh [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+if [[ $deff -ne 0 ]]; then 
+       cp bootcamp_aca.conf /etc/nginx/sites-enabled/default
+fi
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:22]: delete if statement, you can check if the file is zero or not, like
```
if [[ -z $file ]]
```

File: nginx_conf.sh [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+# Checking if the given URL works and if we already have one
+if [[ $response = 200 ]]; then
+	wget $URL
+fi
+
+# checking if we already have the copy as our nginx default conf
+deff=$(diff bootcamp_aca.conf /etc/nginx/sites-enabled/default)
+
+if [[ $deff -ne 0 ]]; then 
+       cp bootcamp_aca.conf /etc/nginx/sites-enabled/default
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:21]: use `mv` instead of `cp`

File: nginx_conf.sh [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+fi
+
+# reloading service
+service nginx reload
+
+# removing the conf file
+rm -f bootcamp_aca.conf
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:28]: delete

File: nginx_conf.sh [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+# checking if we already have the copy as our nginx default conf
+deff=$(diff bootcamp_aca.conf /etc/nginx/sites-enabled/default)
+
+if [[ $deff -ne 0 ]]; then 
+       cp bootcamp_aca.conf /etc/nginx/sites-enabled/default
+fi
+
+# reloading service
+service nginx reload
```
Comment (mkrtchyan-t) [inline nginx_conf.sh:25]: test before reloading, like
```
sudo nginx -t
```

File: nginx_conf.sh (hunk 4 above)
Comment (Frida199) [reply nginx_conf.sh:4]: done

File: nginx_conf.sh (hunk 5 above)
Comment (Frida199) [reply nginx_conf.sh:9]: done

File: nginx_conf.sh (hunk 6 above)
Comment (Frida199) [reply nginx_conf.sh:10]: done

File: nginx_conf.sh (hunk 7 above)
Comment (Frida199) [reply nginx_conf.sh:18]: done

File: nginx_conf.sh (hunk 8 above)
Comment (Frida199) [reply nginx_conf.sh:22]: done

File: nginx_conf.sh (hunk 9 above)
Comment (Frida199) [reply nginx_conf.sh:21]: done

File: nginx_conf.sh (hunk 10 above)
Comment (Frida199) [reply nginx_conf.sh:28]: done

File: nginx_conf.sh (hunk 11 above)
Comment (Frida199) [reply nginx_conf.sh:25]: done

File: print_stat.sh [hunk 12]
```diff
@@ -1,12 +1,21 @@
...
+
+# initiating memory usage
 mem="$(free -m | tr -s " " | cut -d " " -f 3,2 | tr " " "/" | head -2 | tail -1)MB"
- 
+
+# initiating disk usage
 disk=$(df -h | head -4 | tail -1 | tr -s " " | cut -d " " -f 5)
 
+# initiating cpu usage
 cpu=$(top -bn1 | grep load | tr -s " " | cut -d " " -f 12)
```
Comment (mkrtchyan-t) [inline print_stat.sh:14]: should find **CPU usage percentage**, not load average.

File: print_stat.sh [hunk 13]
```diff
@@ -1,12 +1,21 @@
...
 
-echo "Memory Usage: $mem \n Disk Usage: $disk \n CPU Load: $cpu" > /var/www/bootcamp_aca/index.html
+# redirecting statistics to our index.html file
+echo "Memory Usage: $mem \n Disk Usage: $disk \n CPU Usage: $cpu %" > /var/www/bootcamp_aca/index.html
```
Comment (mkrtchyan-t) [inline print_stat.sh:17]: add \<br\> instead of newlines

File: print_stat.sh [hunk 14]
```diff
@@ -1,12 +1,21 @@
...
+# we are using while loop to recompile the code for every 10 seconds
 while true
 do
+
+# initiating memory usage
 mem="$(free -m | tr -s " " | cut -d " " -f 3,2 | tr " " "/" | head -2 | tail -1)MB"
- 
+
+# initiating disk usage
 disk=$(df -h | head -4 | tail -1 | tr -s " " | cut -d " " -f 5)
```
Comment (mkrtchyan-t) [inline print_stat.sh:11]: change completely

File: print_stat.sh (hunk 13 above)
Comment (mkrtchyan-t) [reply print_stat.sh:17]: added

File: print_stat.sh (hunk 14 above)
Comment (mkrtchyan-t) [reply print_stat.sh:11]: changed

**Pull request merged**